cmd/textiled/main.go

```diff
@@ -142,6 +146,12 @@ func init() {
 		flags["addrGatewayUrl"].DefValue.(string),
 		"Public gateway address")
 
+	// Filecoin settings
+	rootCmd.PersistentFlags().String(
+		"addrFilecoinApi",
+		flags["addrFilecoinApi"].DefValue.(string),
+		"Filecoin gRPC API listen address")
```

Comment:
Is it the listen address or just the address? Listen sounds like you're binding to it, where here you're just using it to create a client. Just a personal bias, not important.

Reply:
good call.